src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,164 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> start;
+    private Node<T> end;
```

Comment:
```suggestion
    private Node<T> head;
    private Node<T> tail;
```
you have methods addToTail, so lets call fields ahead and tail accordingly

Reply:
OK